manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -232,6 +240,12 @@ spec:
         - name: shared-doca-driver-toolkit
           emptyDir: {}
         {{- end }}
+        {{- if .RuntimeSpec.UseDriversInventory }}
+        - name: drivers-inventory
+          hostPath:
+            path: /var/opt/ofed-inventory
```

Comment:
i would prefer this to not be flat as this might expand later to other dirs

WDYT about the following path :  `/var/opt/mofed-container/cache`